test/PowerShellEditorServices.Test/Session/ScriptFileTests.cs

```diff
@@ -651,6 +651,7 @@ public void DocumentUriReturnsCorrectStringForAbsolutePath()
         [InlineData("vscode-notebook-cell:/Users/me/Documents/test.ps1#0001", true)]
         [InlineData("https://microsoft.com", true)]
         [InlineData("Untitled:Untitled-1", true)]
+        [InlineData("'a log statement' > 'c:\\Users\\me\\Documents\\test.txt'\r\n", false)]
         public void IsUntitledFileIsCorrect(string path, bool expected) => Assert.Equal(expected, ScriptFile.IsUntitledPath(path));
```

Comment:
See, this test implies that the function is asking "is this path a URI that indicates we're untitled?"

Reply:
Perhaps we should expect this case to be `true` by flipping the logic with `IsWellFormedUri`. That is, if we're given something that is _not_ a URI, assume it's untitled, but I'm not sure that would work for the first two cases.

Reply:
Yeah, doesn't work, since `/Users/me/Documents/test.ps1` is not a URI.